pkg/data/generic.go

```diff
@@ -73,7 +75,14 @@ func NewFromInterface(client dynamic.Interface, opa opa_client.Data, ns types.Re
 	return s
 }
 
-//WithBackoff tunes the values of exponential backoff and jitter factor
+// WithIgnoreRelicas provides a list of namespaces to ignore
+func WithIgnoreRelicas(ignoreNamespaces []string) Option {
```

Comment:
Replicas?

Reply:
Why not call them namespaces here?

Reply:
Yeah sorry i was scoping in adding annotations later. Changing this to Namespaces!